roles/realm_join/tasks/realm_join_ad.yml

```diff
@@ -0,0 +1,116 @@
+---
+- name: Ensure pexpect to answer password
+  become: yes
+  package: name=pexpect state=latest
+  when: ansible_distribution != "CentOS" or ansible_distribution_major_version >= '8'
+
+- name: Install pip to get supported pexpect version
+  become: yes
+  package: "name={{ item }} state=present"
+  loop:
+    - epel-release
+    - python-pip
```

Comment:
I think `name` accepts a list as well and is faster than looping.

Reply:
:+1: yes it indeed does!